### ISSUE TITLE
sam2tsv lists incorrect reference sequence & positions

### ISSUE THREAD

**Issue by dchakro** [lindenb/jvarkit#190]
Hi,

**Duplicate of https://www.biostars.org/p/9493154/** 

Can anyone help me resolve an issue I'm having with [sam2tsv][1].
It is a nifty piece of software but I have been encountering issues with it regarding the numbering of nucleotides it shows for the reference sequence.

Here's what sam2tsv tells me:
![Output of sam2tsv](https://user-images.githubusercontent.com/35454738/136981647-71297f61-c027-40bf-9ffe-f220377c4997.png)

The nucleotide string marked in red `CTGGCCGAGCTAG` is the read reporting the mutation T>A (line #469).
But the reference sequence listed by sam2tsv (green box) doesn't match the read sequence at all. In fact it is correct sequence from the reference fasta, but it is right-shifted by `16` bases. *With other reference files, this number is different, e.g. 20.*

In fact, if I search the sequence `ATGGAGACCCGCT` in my reference sequence it spans `356-368`. In contrast sam2tsv lists these residues in the range `339-352` (as seen in the green box). Off by exactly 

![searching within the reference](https://user-images.githubusercontent.com/35454738/136981743-2fb49247-61a9-4af7-8319-e8238864b602.png)

To summarize:
1) The position listed in the green box (reference), corresponds to the sequence in the red box (read).
2) The reference sequence listed in the green box, is off by 16 bases.

**Information:**
PacBio HiFi CCS reads aligned using `pbmm2` to custom reference (cDNA). 

**Files:**

[Reference.fa][2]. 

[relevant sam2tsv output][3]

**Command used:**

```
java -jar '~/Git/Others_Cloned/jvarkit/dist/sam2tsv.jar' -R ../../../reference/pBG-ERBB2/ERBB2.fa 1.ERBB2_Library.bam
```

  [1]: https://lindenb.github.io/jvarkit/Sam2Tsv.html
  [2]: https://bin.snopyta.org/?f5a79557297385cd#F8oPqeZBHS6N9QxCrQbuQ4fyW2EfdBv9YkUjb1pEyq3S
  [3]: https://bin.snopyta.org/?59a65086abb1b542#BG4WmD2UW5am1vNsFVbSjWfKQ2hmNe6THUVTTWBS7PFp
